book_source/03_topical_pages/03_pecan_xml.Rmd

```diff
@@ -636,6 +636,42 @@ This information is currently used by the following PEcAn workflow functions:
 - `PEcAn.<MODEL>::write.configs.<MODEL>` -- See [above](#pecan-write-configs)
 - `PEcAn.uncertainty::run.sensitivity.analysis` -- Executes the uncertainty analysis
 
+#### Coordinating inputs with the `input_design` design matrix {#xml-input-design}
+
+Multi-site ensembles and sensitivity analyses that sample over input files use
+an `input_design` data.frame to keep parameter draws and input files aligned
+across runs. The design is created up front (typically via
+`generate_joint_ensemble_design()` inside `runModule.run.write.configs()`) and
+passed through to `run.write.configs()`, `write.ensemble.configs()`, and
+`write.sa.configs()`. It is not saved automatically to `samples.Rdata`, so keep
```

Comment:
AFAIK write.sa.configs doesn't currently use the input design, though we've discussed a future direction of pulling out the SA design code into the design function and then merging write.ensemble.configs & write.sa.configs, but that might not happen for months.